boards/acd52832/include/periph_conf.h

```diff
@@ -104,7 +104,8 @@ static const i2c_conf_t i2c_config[] = {
     {
         .dev = NRF_TWIM1,
         .scl = 28,
-        .sda = 29
+        .sda = 29,
+        .speed = I2C_SPEED_NORMAL
```

Comment:
As you said, this one is actually more than a warning as I2C_SPEED_NORMAL is actually not 0. Thanks cpp check :)